truffle.js

```diff
@@ -2,6 +2,39 @@ require("babel-register");
 require("babel-polyfill");
 const TestRPC = require("ethereumjs-testrpc");
 
+/**
+  In order to deploy normally without this setup you will have to comment all the code from =====>
+*/
+const Web3 = require("web3");
```

Comment:
can you put this intialization subroutine into /scripts as a function and then import it using `require`?